apps/dashboard/lib/collections/deploy/projects.ts

```diff
@@ -11,6 +11,14 @@ const schema = z.object({
   gitRepositoryUrl: z.string().nullable(),
   updatedAt: z.number().int().nullable(),
   liveDeploymentId: z.string().nullable(),
+  // Flattened deployment fields for UI
+  commitTitle: z.string(),
+  branch: z.string(),
+  author: z.string(),
+  commitTimestamp: z.number().int().nullable(),
+  regions: z.array(z.string()),
+  // Domain field
+  domain: z.string(),
```

Comment:
this can have multiple domains

where are cpu and memory?

Reply:
As I said above we only need 1 domain for Project listing UI
<img width="1022" height="272" alt="image" src="https://github.com/user-attachments/assets/20b25a7d-6aba-4e49-a532-f2a12111fba2" />

Reply:
yeah, fair
I want to reuse the same data in other places though, and there we need multiple domains

closing this one in favour of the other thread